src/Main.java

```diff
@@ -1,15 +1,123 @@
-import Employee.EmployeeBook;
+import Employee.*;
+
+import java.text.NumberFormat;
 
 public class Main {
     public static void main(String[] args) {
 
         EmployeeBook EB = new EmployeeBook();
```

Comment:
```suggestion
        EmployeeBook eb = new EmployeeBook();
```
всегда camelCase для названия переменных